problemsets-mat223/modules/module3-exercises.tex

```diff
@@ -1,10 +1,64 @@
 \begin{exercises}
+		% Topics:
+		% Spans
+		% Representing lines/planes/volumes as spans
+		% translated spans
+		% linear independence
 	\begin{problist}
-		\prob  Example problem with parts
+		\prob
+		Find the spans of the following sets of vectors.
```

Comment:
-> `For each set below, determine whether its span is a point, line, plane, volume, or other`